internal/manifest/os.go

```diff
@@ -501,6 +501,8 @@ func (p *OS) serialize() osbuild.Pipeline {
 			commands = []string{fmt.Sprintf("/usr/bin/rhc connect -o=%s -a=%s --server %s", p.Subscription.Organization, p.Subscription.ActivationKey, p.Subscription.ServerUrl)}
 			// insights-client creates the .gnupg directory during boot process, and is labeled incorrectly
 			commands = append(commands, "restorecon -R /root/.gnupg")
+			// execute the rhc post install script as the selinuxenabled check doesn't work in the buildroot container
+			commands = append(commands, "/usr/sbin/semanage permissive --add rhcd_t || true")
```

Comment:
Will `|| true` work? Is the command executed in shell?

Reply:
Ugh of course, I'll just remove that part